app/services/FileTransferService.scala

```diff
@@ -64,6 +70,21 @@ class FileTransferService @Inject()(
   //    )
   //  }
 
+  private def batchTransfer(caseId: String, conversationId: String, files: Seq[SupportingDocument])
+                           (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = {
+    val correlationId = UUID.randomUUID().toString
+    val request = MultiFileTransferRequest.fromSupportingDocuments(
+      caseReferenceNumber = caseId,
+      conversationId = conversationId,
+      correlationId = correlationId,
+      applicationName = "C18",
+      uploadedFiles = files
+    )
+    connector.transferMultipleFiles(request).map { _ =>
+      // todo: audit the response
+    }
```

Comment:
should this be fire-and-forget like the other function? (it'll make the caller wait until the upload is complete)

Reply:
actually, it was fire-and-forget anyway, because the Future is not waited on in the service:
https://github.com/hmrc/import-voluntary-disclosure-submission/blob/master/app/services/UpdateCaseService.scala#L35
so this will not prevent the request from completing